From 4faab2add28236e83b1908aeb38a00ab37ea2af0 Mon Sep 17 00:00:00 2001
From: Josef Schlehofer <pepe.schlehofer@gmail.com>
Date: Sat, 15 Oct 2022 15:36:00 +0200
Subject: [PATCH] mpc85xx: fix missing kernel config symbol

While compiling OpenWrt master for Turris 1.x routers (p2020), it
reported following error:

    Gianfar Ethernet (GIANFAR) [Y/n/m/?] y
    Freescale DPAA2 Ethernet Switch (FSL_DPAA2_SWITCH) [N/m/y/?] (NEW)
Error in reading or end of file.

Let's fix it by disabling it.

Signed-off-by: Josef Schlehofer <pepe.schlehofer@gmail.com>
---
 target/linux/mpc85xx/config-5.15 | 1 +
 1 file changed, 1 insertion(+)

diff --git a/target/linux/mpc85xx/config-5.15 b/target/linux/mpc85xx/config-5.15
index 4f27d3bd70..8734038d33 100644
--- a/target/linux/mpc85xx/config-5.15
+++ b/target/linux/mpc85xx/config-5.15
@@ -71,6 +71,7 @@ CONFIG_ENABLE_MUST_CHECK=y
 CONFIG_ETHERNET_PACKET_MANGLE=y
 CONFIG_FIXED_PHY=y
 CONFIG_FSL_BOOKE=y
+# CONFIG_FSL_DPAA2_SWITCH is not set
 CONFIG_FSL_EMB_PERFMON=y
 # CONFIG_FSL_FMAN is not set
 CONFIG_FSL_LBC=y
-- 
2.34.1

